announcement.html

```diff
@@ -34,8 +34,32 @@
     <meta property="og:type" content="website">
     
     <script async defer src="https://buttons.github.io/buttons.js"></script>
+
+    <!-- Dynamically Update Github Stars Count -->
+    <script>
+      function updateStarCount() {
+          const repoUrl = "https://api.github.com/repos/opentffoundation/manifesto";
+
+          fetch(repoUrl)
+              .then(response => response.json())
+              .then(data => {
+                  const starCount = data.stargazers_count;
+                  let formattedStarCount;
+
+                  if (starCount >= 1000) {
+                      formattedStarCount = (starCount / 1000).toFixed(1) + "k";
+                  } else {
+                      formattedStarCount = starCount.toString();
+                  }
+
+                  const starElement = document.getElementById("star-count");
+                  starElement.textContent = formattedStarCount;
+              })
```

Comment:
```suggestion
                  const starCount = data.stargazers_count;
                  const formattedStarCount = (starCount / 1000).toFixed(1) + "k";

                  document.getElementById("star-count").textContent = formattedStarCount;
              })
```

I'd do it like this, because it is a lot more consice now. Also, the stars won't drop below 1k since they are at 10k already